io/plugins/eu.esdihumboldt.hale.io.wfs.ui/src/eu/esdihumboldt/hale/io/wfs/ui/getfeature/GetFeatureParamsPage.java

```diff
@@ -60,6 +66,18 @@ public boolean updateConfiguration(WFSGetFeatureConfig configuration) {
 		else {
 			configuration.setMaxFeatures(null);
 		}
+
+		if (resolveLinks.getSelection()) {
+			if (resolveLinksRadioValue.getSelection()) {
+				// Set resolve depth based on the value from the spinner
+				int selectedDepth = resolveLinksValueSpinner.getSelection();
+				configuration.setResolvedepth(String.valueOf(selectedDepth));
+			}
+			else if (resolveLinksRadioAllValues.getSelection()) {
+				// Set resolve depth to a predefined value for all values
+				configuration.setResolvedepth(resolveDepthAll);
+			}
+		}
```

Comment:
I think in case the configuration object was manipulated before, in the else case the value should be set to `null`.

Reply:
Where could that object be manipulated before?
This is just for the case when we want to look in all the depth...so '*'.

If I set the parameter here to null, then I should check also in the places where is used if the parameter is null, instead done as it is I can just use the parameter as it is. And this comes also from the UI where the user sees a '*' and they know what is it.

Reply:
I mean the else case for this block, i.e. if `resolveLinks.getSelection()` is `false`.